verify/db.go

```diff
@@ -73,20 +73,20 @@ func (db *DB) AddKey(id string, k *data.Key) error {
 	return nil
 }
 
-var validRoles = map[string]struct{}{
+var topLevelRoles = map[string]struct{}{
 	"root":      {},
 	"targets":   {},
 	"snapshot":  {},
 	"timestamp": {},
 }
 
-func ValidRole(name string) bool {
-	_, ok := validRoles[name]
+func IsTopLevelRole(name string) bool {
```

Comment:
This is a breaking change as ValidRole is exportable, to avoid breaking and keep your readability improvement you could:
1. turn IsTopLevelRole private to avoid adding new possibilites
2. keep ValidRole

Reply:
Good catch -- fixed this.